docs/source/tutorials.rst

```diff
@@ -1,82 +1,55 @@
-The behavior describing how the robot drives around in the room is modeled as a Deterministic Markov Chain (DTMC) shown in the picture below. In each step, the robot moves forward in 50% of the cases and rotates in 50% of the cases. In case it bumps into a wall, it just stops at the collision point and continues operating from there. What is omitted in the picture is the calculation of this collision point and the conversion to and from floats to integers. The latter is only necessary to make the example run in Storm because the tool currently does not support transient floats.
-
-.. image:: graphics/dtmc.PNG
+.. image:: graphics/scxml_tutorial_ros_example_w_bt.drawio.svg
     :width: 800
-    :alt: An image of the DTMC representing the robot's behavior.
-
-The property given in the JANI file checks for the minimal probability that eventually within 10 000 steps the position (1.0, 1.0) is reached with an error range of 0.05 m.
+    :alt: An image of the complete exemplary system.
 
+In this example, the system is composed by the following components modeled in SCXML:
 
+* a **battery_drainer**, that at each time step drains the battery by 1%, and each time the charge trigger is received, it recharges the battery to 100%.
+* a **battery_manager**, that at each time the battery level is received checks if it is below 30% and, if so, triggers the alarm.
 
+The **behavior tree** continuously checks the alarm topic and, once it is triggered, sends a charge trigger to the battery_drainer.
 
+The JANI property given in `battery_depleted.jani <https://github.com/convince-project/as2fm/tree/main/jani_generator/test/_test_data/ros_example/battery_depleted.jani>`_ defines the property of interest to be model checked. In this case, it calculates the minimal probability that the battery level is below or equal to zero eventually, i.e., all we verify here is that the battery is empty at some point.
 
-
-How to convert from (SC)XML to plain JANI?
---------------------------------------------
-
-But writing a JANI model by hand is quite difficult. Therefore we also developed an approach to directly extract a JANI model from the robotic system specified in (SC)XML files, e.g., for the ROS nodes, the environment, the behavior tree, and the interaction of those components. 
-
-Running the script
-`````````````````````
-
-A full system model can be converted into a model-checkable JANI file as follows.
-
-.. code-block:: bash
-
-    scxml_to_jani path_to_main.xml
+In the `main.xml file <https://github.com/convince-project/as2fm/tree/main/jani_generator/test/_test_data/ros_example/main.xml>`_ introduced earlier, the maximum run time of the system is specified with ``max_time`` and shared across the components. To make sure that the model checked property makes sense, the allowed runtime needs to be high enough to have enough time to deplete the battery, i.e., in this example the maximal time needs to be at least 100s because the battery is depleted by 1% per second.
```

Comment:
This is maybe an aspect that goes a little too deep for a first tutorial. I mean we probably need a how-to for properties at some point anyway. Maybe we can mention this there? But just an idea. We can also keep it here for now.

Reply:
Maybe we can keep it as is, and point to some other reference in the how to for further info?